dashboard/app/bisect_test.go

```diff
@@ -152,7 +152,7 @@ Date:   Wed Feb 9 04:05:06 2000 +0000
     kernel: add a bug
 
 bisection log:  %[2]v
-start commit:   11111111 kernel_commit_title1
+oldest commit:   11111111 kernel_commit_title1
```

Comment:
For successful bisection this is not "oldest commit", this is "start commit" of bisection process, no?